src/Component.php

```diff
@@ -2,15 +2,53 @@
 
 declare(strict_types=1);
 
-namespace MyComponent;
+namespace Keboola\StorageWriter;
 
 use Keboola\Component\BaseComponent;
+use Keboola\Component\UserException;
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
 
 class Component extends BaseComponent
 {
     public function run(): void
     {
-        // @TODO implement
+        /** @var Config $config */
+        $config = $this->getConfig();
+        $client = new Client(['token' => $config->getToken(), 'url' => $config->getUrl()]);
+        $tokenInfo = $client->verifyToken();
+        if (!$tokenInfo['bucketPermissions']) {
```

Comment:
v `bucketPermissions` se vracejí vždy všechny buckety na které má token práva. Pokud je to prázdné znamená to že nemá právo na žádný bucket. Myslím že tahle podmínka je tady zbytečná.